### PR TITLE
skipper: update canary version to v0.21.84

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
[Changes](https://github.com/zalando/skipper/compare/v0.21.76...v0.21.84)

* https://github.com/zalando/skipper/pull/3060
* https://github.com/zalando/skipper/pull/3061
* https://github.com/zalando/skipper/pull/3062
* https://github.com/zalando/skipper/pull/3063
* https://github.com/zalando/skipper/pull/3064
* https://github.com/zalando/skipper/pull/3057
* https://github.com/zalando/skipper/pull/3055
* https://github.com/zalando/skipper/pull/3053

Also configures required min-drop-probability for passive health checker.

### CODE REVIEW

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by RomanZavodskikh** [comment]
> https://github.com/zalando/skipper/pull/3053

You must include `-min-drop-probability` option.

**Comment by RomanZavodskikh** [comment]
See https://github.com/zalando/skipper/blob/v0.21.84/docs/operation/operation.md#passive-health-check-experimental

Add `min-drop-probability=0.05` here https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/config-defaults.yaml#L82 .

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ skipper_ingress_max_replicas: "50"
 {{end}}
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1500Mi"
-skipper_ingress_health_check_options: "period=10s,min-requests=10,max-drop-probability=0.9"
+skipper_ingress_health_check_options: "period=10s,min-requests=10,min-drop-probability=0.05,max-drop-probability=0.9"
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:82]: @RomanZavodskikh noted this will fail for non-canary.

Comment (AlexanderYastrebov) [reply cluster/config-defaults.yaml:82]: e2e failed as expected:
```
message: back-off 5m0s restarting failed container=skipper-ingress pod=skipper-ingress-7bc5947c75-z7cfs_kube-system(3ceafbb0-546c-42e7-ac35-c5e0a219e8d5)
                    reason: CrashLoopBackOff
```

**Comment by AlexanderYastrebov** [comment]
Will open a new PR as we decided to add default value for `min-drop-probability` https://github.com/zalando/skipper/pull/3067

**Pull request closed without merging**